hw/ip/aes/data/aes.hjson

```diff
@@ -192,6 +193,7 @@
         The order in which the registers are updated does not matter.
         Can only be updated when the AES unit is idle.
         If the AES unit is non-idle, writes to these registers are ignored.
+        Upon reset, these registers are cleared with pseudo-random data.
```

Comment:
... cleared with pseudo ... Perhaps "initialized" is a better word, here and below.

Reply:
Please refer to the [other discussion](https://github.com/lowRISC/opentitan/pull/5651/files#r595892421).